app/webhook.go

```diff
@@ -159,11 +159,37 @@ func CreateWebhookPost(userId, teamId, channelId, text, overrideUsername, overri
 		}
 	}
 
-	if _, err := CreatePost(post, teamId, false); err != nil {
-		return nil, model.NewLocAppError("CreateWebhookPost", "api.post.create_webhook_post.creating.app_error", nil, "err="+err.Message)
+	splits := make([]string, 0)
+	remainingText := post.Message
+
+	for len(remainingText) > model.POST_MESSAGE_MAX_RUNES {
+		splits = append(splits, remainingText[:model.POST_MESSAGE_MAX_RUNES])
+		remainingText = remainingText[model.POST_MESSAGE_MAX_RUNES:]
+	}
+
+	splits = append(splits, remainingText)
+
+	var firstPost *model.Post = nil
+
+	for _, txt := range splits {
+		post.Id = ""
+		post.UpdateAt = 0
+		post.CreateAt = 0
+		post.Message = txt
+		if _, err := CreatePost(post, teamId, false); err != nil {
```

Comment:
Ok cool and anything after that is in a new post object from what the store returns